track.go

```diff
@@ -32,9 +33,11 @@ type Track struct {
 
 	packetizer rtp.Packetizer
 
-	receiver         *RTPReceiver
-	activeSenders    []*RTPSender
-	totalSenderCount int // count of all senders (accounts for senders that have not been started yet)
+	receiver             *RTPReceiver
+	activeSenders        []*RTPSender
+	totalSenderCount     int // count of all senders (accounts for senders that have not been started yet)
+	totalSamplesSent     uint32
```

Comment:
lets use atomic operations for these `total*` properties rather than holding the mutex

https://golang.org/pkg/sync/atomic/